spec/integration/balancing_spec.rb

```diff
@@ -167,8 +172,10 @@
     click_link I18n.t('finance.balancing.edit_results_by_articles.add_article')
     expect(page).to have_selector('form#new_order_article')
     within('#new_order_article') do
-      select new_article.name, :from => 'order_article_article_id'
+      find('#order_article_article_id').select(new_article.name)
+      sleep 0.1
```

Comment:
i don't think that i need to mention that `sleep` looks wrong :wink:

Reply:
thanks for mentioning, I'd like to look into this sometime
https://github.com/twalpole/apparition#timing-problems
https://github.com/teamcapybara/capybara#asynchronous-javascript-ajax-and-friends